pkg/graph/schema.resolvers.go

```diff
@@ -722,6 +722,29 @@ func (r *queryResolver) AccessibilityRequests(ctx context.Context, after *string
 	return &model.AccessibilityRequestsConnection{Edges: edges}, nil
 }
 
+func (r *queryResolver) Requests(ctx context.Context, after *string, first int) (*model.RequestsConnection, error) {
+	requests, queryErr := r.store.FetchMyRequests(ctx)
```

Comment:
This would be more reusable if it were `FetchRequestsByEUAID` but I do like how clean this is. Do we expect to have a situation where, for example, the 508 tteam needs to fetch all the requests belonging to a given user? Even if we do expect that case, is it worth keeping this method like this for the succinctness?